### ISSUE TITLE
Add support for `0x50` firstByte

### ISSUE THREAD

**Issue by mgcrea** [SheetJS/sheetjs#99]
I've got `.xlsx` files that are rejected by the lib `Unsupported file`. Any idea what I should start looking for? Tried to add:

``` js
case 0x50: return parse_xlml(d, (o.type="buffer",o));
```

But it breaks at:

``` js
node_modules/xlsjs/xls.js:6366
            switch(state[state.length-1][0]) {
TypeError: Cannot read property '0' of undefined
    at parse_xlml_xml (node_modules/xlsjs/xls.js:6366:32)
```

Great job by the way. Would love to switch all future developments to a lib with a "clean" license.

---

Found [this](http://stackoverflow.com/questions/19792056/detection-password-protected-ms-office-files-in-c-c) stackoverflow answer:

``` js
If the files are in Office 2007 format (e.g. .docx), then their internal storage is either:

1) A zip file of xml docs (if it's not password protected)

2) The old style compound file format (if it IS password protected).

Therefore you could probably do something like this:

1) Check the first few bytes of the file

2) If it's a zip file (non password protected), it'll start with 0x50 0x4b 0x03 0x04.

3) If it's not a zip file, then it's probably password protected. It will start with a different binary signature (e.g. Word 2007 docs start with 0xd0 0xcf 0x11 0xe0 in this case)

Basically, if it's a new .docx or .xlsx, and it DOESN'T start with the zip signature of 0x50 0x4b 0x03 0x04, it's probably password protected.

For other versions of MS Office, it's a bit trickier...
```

So it looks like it's a standard zip file.
